### DISCUSSION TITLE
Separating Lead Guitar and Rhythm Guitar?

### DISCUSSION THREAD

**Discussion by abner4n** [Anjok07/ultimatevocalremovergui#1226] [Q&A]
Is there an algo tha can be used to separate lead guitar from a mix that have other guitars too?

**Comment by rsuh** [comment]
also wondering thiss

**Comment by rschader** [comment]
I'm searching for an answer to this also! I ran my first separation test on Mean Town Blues from Johnny Winter And Live, and it did a great job separating the vocals, drums and bass, then left me with another track named Other that contained to 2 guitar parts together. Thinking of trying to run that other track thru the process again to see if it will separate them? Maybe it would help if we could tell the program how many instruments there should be? Is that an option I'm not aware of yet (I just installed this today)?
